packages/ui/src/context/CartProvider.tsx

```diff
@@ -535,6 +537,17 @@ function cartStore({
         const tokens: Token[] = []
         const ordersToFetch: string[] = []
 
+        const tokensByMaker = updatedItems.reduce((map, item) => {
```

Comment:
maybe we should call this `1155sByMaker` to make it clear that it does not include all tokens

Reply:
going to keep it as it's more than just 1155s